src/Prometheus.Client/ThreadSafeDouble.cs

```diff
@@ -6,7 +6,7 @@
 [assembly: InternalsVisibleTo("Prometheus.Client.Tests" +
```

Comment:
Why we have assembly attribute in such "hidden" place? Should we move it in assemblyInfo.cs?